src/radical/pilot/agent/radical-pilot-agent-multicore.py

```diff
@@ -2038,7 +2045,7 @@ def construct_command(self, task_exec, task_args, task_numcores,
         hosts_string = ",".join([slot.split(':')[0] for slot in task_slots])
         export_vars  = ' '.join(['-x ' + var for var in self.EXPORT_ENV_VARIABLES if var in os.environ])
 
-        orte_command = '%s --hnp "%s" %s -np %s -host %s %s' % (
+        orte_command = '%s --debug-devel --hnp "%s" %s -np %s -host %s %s' % (
             self.launch_command, dvm_uri, export_vars, task_numcores, hosts_string, task_command)
```

Comment:
debug is  synced with the debug level on module side:
https://github.com/radical-cybertools/radical.pilot/blob/devel/src/radical/pilot/controller/pilot_launcher_worker.py#L616